pkg/monitor/sqsevent/sqs-monitor.go

```diff
@@ -176,6 +203,10 @@ func (m SQSMonitor) processInterruptionEvents(interruptionEventWrappers []Interr
 			log.Warn().Err(eventWrapper.Err).Msg("dropping interruption event for an already terminated node")
```

Comment:
This looks like a slightly different way of handling messages we want to skip (direct reference to a particular error object `ErrNodeStateNotRunning`). Your new solution of the `skip` struct seems more extensible if we encounter similar events in the future. Should we (perhaps in a future PR) refactor `ErrNodeStateNotRunning` to use the new `skip` struct?